lib/ansible/plugins/doc_fragments/constructed.py

```diff
@@ -47,7 +47,7 @@ class ModuleDocFragment(object):
         - The key from input dictionary used to generate groups.
       default_value:
         description:
-        - The default value when the host variable's value is an empty string.
+        - The default value when the host variable's value is V(None) or an empty string.
```

Comment:
Can you update the documentation for trailing_separator too?

Reply:
Has been updated, thanks for the pickup.

Reply:
also add note that this is new behavior and the version at which it starts